_posts/2021-09-03-normalization-of-EJB-principal-propagation.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: 'Normalize principal propagation/injection across elytron and legacy'
+date: 2021-09-03
```

Comment:
I think this will need to be changed to 2021-03-09 both here and in the title.

Reply:
@fjuma Thank you, fixed.